packages/vue3/src/components/Spinner/Spinner.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import type { SpinnerType } from './types';
+
+defineProps<{ spinner: SpinnerType }>();
+</script>
+
+<template>
+  <div class="container">
+    <div class="spinner" />
+    <p class="label">
+      {{ spinner.label }}
+    </p>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.container {
+  display: flex;
+  flex-direction: column;
+}
+.label {
+  font: v-bind(
+    "`var(--text-${spinner.labelFontSize}-regular)` || 'var(--text-md-regular)'"
```

Comment:
We can use `withDefaults` in the props definitions so that we don't have to use the `||` operator here